src/Maintenance/Maintenance/help/Az.Maintenance.md

```diff
@@ -0,0 +1,43 @@
+---
+Module Name: Az.Maintenance
+Module Guid: {{ Update Module Guid }}
```

Comment:
Update this with https://github.com/Azure/azure-powershell/pull/10313/files#diff-9b895a4cb333f6bf9176b1eeb9260782R21